test/test.js

```diff
@@ -77,7 +77,7 @@ describe('Operator names: ', function () {
   });
 
   it('should return MPT', function () {
-    var opName = myanmarPhoneNumber.getTelecomName('09420012345');
+    var opName = myanmarPhoneNumber.getTelecomName('959890743931');
     assert.equal(opName, 'MPT');
```

Comment:
```suggestion
    var opName1 = myanmarPhoneNumber.getTelecomName('09420012345');
    var opName2 = myanmarPhoneNumber.getTelecomName('959890743931');
    assert.equal(opName1, 'MPT');
    assert.equal(opName2, 'MPT');
```

Could you retain old test cases like this?